src/Tizen.NUI/src/internal/EXaml/Operation/CreateInstance.cs

```diff
@@ -57,7 +58,13 @@ public void Do()
                         {
                             paramList[i] = globalDataList.GatheredInstances[(paramList[i] as Instance).Index];
                         }
+
+                        if (paramList[i] is CombinedString)
+                        {
+                            paramList[i] = (paramList[i] as CombinedString).RealString;
+                        }
```

Comment:
I think it can be changed as the below, I do you think? 

if (paramList[i] is CombinedString value)
{
  paramList[i] = value.RealString;
}